webapp-django/crashstats/crashstats/tests/test_models.py

```diff
@@ -1642,3 +1628,19 @@ def mocked_reprocess(crash_id):
         # Note that it doesn't raise an error if
         # the ReprocessingOneRabbitMQCrashStore choses NOT to queue it.
         ok_(not api.post(crash_id='bad-crash-id'))
+
+    def test_Priorityjob(self):
+        api = models.Priorityjob()
+
+        def mocked_process(crash_id):
+            if crash_id == 'some-crash-id':
+                return True
+            elif crash_id == 'bad-crash-id':
+                return
+            raise NotImplementedError(crash_id)
+
+        models.Priorityjob.implementation().process = mocked_process
+        ok_(api.post(crash_id='some-crash-id'))
+        # Note that it doesn't raise an error if
+        # the PriorityjobRabbitMQCrashStore choses NOT to queue it.
+        ok_(not api.post(crash_id='bad-crash-id'))
```

Comment:
I believe this test shows something is wrong, or is not testing anything. Your ``Priorityjob`` model requires the ``crash_ids`` argument, and it's missing in both cases here.